config.env.example

```diff
@@ -3,11 +3,16 @@ DEYE_LOGGER_PORT=8899
 DEYE_LOGGER_SERIAL_NUMBER=1234567890
 
 MQTT_HOST=192.168.x.x
-MQTT_PORT=1883
-MQTT_USERNAME=user
-MQTT_PASSWORD=password
-MQTT_TOPIC_PREFIX=deye
+# MQTT_PORT=1883
+# MQTT_USERNAME=user
+# MQTT_PASSWORD=password
+# MQTT_TOPIC_PREFIX=deye
+# MQTT_TLS_ENABLED=true
+# MQTT_TLS_INSECURE=true
+# MQTT_TLS_CA_CERT_PATH=certs/ca.crt
+# MQTT_TLS_CLIENT_CERT_PATH=certs/client.crt
+# MQTT_TLS_CLIENT_KEY_PATH=certs/client.key
 
-LOG_LEVEL=INFO
-DEYE_DATA_READ_INTERVAL=60
+# LOG_LEVEL=INFO
+# DEYE_DATA_READ_INTERVAL=60
```

Comment:
Why the properties are commented out now?

Reply:
As per my experience it is typical to comment out settings which have a default value. This helps the user to focus on the settings which are mandatory for the code.